pages/common/git-contrib.md

```diff
@@ -0,0 +1,9 @@
+# git contrib
+
+> Display commits from a specific author.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-contrib>.
+
+- Display all commit hashes and corresponding commit messages from a specific author/username:
+
+`git contrib {{user_name}}`
```

Comment:
`username` is a word:
```suggestion
`git contrib {{username}}`
```

Reply:
But is it referring to e.g. the GitHub handle (`navarroaxel`) or the user's display name (`Axel Navarro`)?

Reply:
Based on the `git log --author` you can use name, full name, username or email.